usr/lib/python3/dist-packages/linuxmusterLinuxclient7/logging.py

```diff
@@ -93,6 +99,10 @@ def printLogs(compact=False):
                 if compact:
                     # "^([^ ]+[ ]+){4}" matches "Apr  6 14:39:23 somehostname" 
                     line = re.sub("^([^ ]+[ ]+){4}", "", line)
+                if anonymize and isConfigOk:
+                    line = re.sub(serverHostname, "server.linuxmuster.example.com", line)
```

Comment:
Please use `linuxmuster.lan` as a generic domain

Reply:
well, thought so at first. But then again: that might be the default in many systems and so you don't know if it was anonymized or not, so why not `linuxmuster.anonymized.lan` ?

Reply:
If you are using linuxmuster.lan, it does not matter if it's anonymized in my opinion. The hostname ist not reachable from outside ayway. It would be like anonymizing 10.0.0.1

Reply:
Yes, I know. What I mean: If you or somebody else is trying to help on the forum or otherwise, you cannot be sure, if `linuxmuster.lan` is anonymized or just the regular domain. You would have to ask again, just to make sure the person does not have some weird domain configured which might be a reason for the problems he/she has.

Reply:
Anyway, you can just resolve and merge as is.

Reply:
Ahhh, I see. That's a valid point. How about replacing all letters in the domain with `X`? This would also reveal wiered legths and special chars.

Reply:
Eg. With a regex like `[a-zA-Z]`

Reply:
This may be overkill. This is fine for a first version.